<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="endScanButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="webLayout">
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>endScanButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>endScanButtonEvent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>nextButtonEvent()</slot>
  <slot>endScanButtonEvent()</slot>
 </slots>
</ui>
